src/System.IO.Pipelines/PipeFactory.cs

```diff
@@ -49,9 +49,9 @@ public IPipeReader CreateReader(Stream stream)
             return pipe;
         }
 
-        private async void ExecuteCopyToAsync(Pipe pipe, Stream stream)
+        private Task ExecuteCopyToAsync(Pipe pipe, Stream stream)
         {
-            await stream.CopyToAsync(pipe);
```

Comment:
It can't throw synchronously here though so not an issue. It's inside of an async method.